build_tools/ci/cpu_comparison/input_generator.py

```diff
@@ -218,21 +239,5 @@ def generate_inputs(filename, write_dir, seed):
         )
 
     command_flags = "  ".join(input_args)
-    command_arg_filename = os.path.join(write_dir, name + "_input_args.txt")
-    with open(command_arg_filename, "w") as file:
-        file.write(command_flags)
-
+    return command_flags
 
-if __name__ == "__main__":
```

Comment:
These "if __name__" bits of code are no longer needed, as the python scripts are no longer called from bash